.build/build-owasp.xml

```diff
@@ -16,9 +16,11 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<project basedir="." name="apache-cassandra-owasp-tasks">
+<project basedir="." name="apache-cassandra-owasp-tasks"
+         xmlns:if="ant:if"
+         xmlns:unless="ant:unless">
     <property name="dependency-check.version" value="8.3.1"/>
-    <property name="dependency-check.home" value="${build.dir}/dependency-check-ant-${dependency-check.version}"/>
+    <property name="dependency-check.home" value="${user.home}/.ant/dependency-check-ant-${dependency-check.version}"/>
```

Comment:
I set it to user.home because downloading the database is painfully slow and it can be shared between projects / worktrees. Redownloading it after ant realclean was frustrating

Reply:
is it specific to `ant` ?  

could it be under `${user.home}/.owasp/dependency-check-ant-${dependency-check.version}` instead?

how big does it get? should it go under `${user.home}/.cache/owasp/…` instead?  (mine was ~500M , but it will grow over time…)

@smiklosovic thoughts?

Reply:
I'm fine with any location under user home. It takes ~250M currently

Reply:
@michaelsembwever maybe there is some public server which hosts such database and we could connect to it instead (or at least as a first choice)?

Reply:
@jacek-lewandowski what happens if `~/.cache` dir does not exist? Highly improbable but still ... does it create parent dirs too?

Reply:
yes, it creates the parent directories as well, I've set it to `~/.cache/dependency-check-ant/owasp-db`

Reply:
i agree with @smiklosovic , `~/.cache/cassandra-owasp-dependency-check` is a better name.

Users will want to know where a half gigabyte folder came from.